src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -411,7 +402,7 @@ internal bool IsPartialImplementation
         {
             get
             {
-                return this.IsPartial && (_hasAnyBody || HasExternModifier);
+                return this.IsPartial && (flags.HasAnyBody || HasExternModifier);
```

Comment:
>flags.HasAnyBody [](http://example.com/codeflow?start=42&length=16)

```HasAnyBody```? I think we should prefer existing members over accessing flags directly.

Reply:
sure. :)